extensions/dicom-sr/src/tools/DICOMSRDisplayTool.js

```diff
@@ -1,4 +1,5 @@
 import { importInternal, getToolState, toolColors } from 'cornerstone-tools';
+import { pixelToCanvas } from 'cornerstone-core';
```

Comment:
All of this makes sense. Makes me wonder if there's a nice way to add this to our monorepo that publishes this tool as a separate package. `@ohif/cstool-dicomsr`

Reply:
I don't think the tool has much value without the viewport, as it needs something to set up what is tracked per element.